pymzml/pymzML
# Update CI/CD

- start tests with tox
  - test python 3.5 to 3.7
  - test building docs
  - run example_scripts
  - optionally check coding style
- add development requirements
- setup.py reads requirements and installs them with setup.py install/develop
make development compatible with tox
- make travis autodeploy when new tag is set
- update plotting so pymzML makes use of the latest ploly version
